feeds/wlan-ap/opensync/patches/19-events-protobuf-definitions.patch

```diff
@@ -0,0 +1,225 @@
+Index: opensync-2.0.5.0/interfaces/opensync_stats.proto
```

Comment:
https://github.com/Telecominfraproject/wlan-ap/pull/72 looks like the same patch included int hat PR ? I merged #72, shall I drop it and use this instead ?

Reply:
Yes, I had a discussion with Rick about that. Drop the #72 .

Reply:
yes that is the common pattern used for this API